### PR TITLE
Network query

### PR DESCRIPTION
**Pull request by geofjamg**

### CODE REVIEW

File: afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java [hunk 1]
```diff
@@ -60,82 +58,88 @@ private String getScriptOutput() {
...
-            scriptOutputWriter.flush();
-            scriptOutput = scriptOutputWriter.toString();
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
-
-        if (scriptError == null) {
-            return new ModifiedNetwork(network, null, previousScriptOutput + scriptOutput);
+        ScriptResult result = ScriptUtils.runScript(network, script.getScriptType(), script.readScript());
+        if (result.getError()  == null) {
```
Comment (mathbagu) [inline afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java]: Remove extra space before ==

File: afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java [hunk 2]
```diff
@@ -60,82 +58,88 @@ private String getScriptOutput() {
...
-    private ModifiedNetwork loadNetwork(VirtualCase virtualCase) {
-        try {
-            return cache.get(virtualCase.getId(), () -> {
-                ProjectCase baseCase = virtualCase.getCase();
+    private ModifiedNetwork loadNetworkFromVirtualCase(VirtualCase virtualCase) {
+        ProjectCase baseCase = virtualCase.getCase();
 
-                // getNetwork network
-                Network network = baseCase.getNetwork();
+        // getNetwork network
```
Comment (mathbagu) [inline afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java]: This comment is strange

File: afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java [hunk 3]
```diff
@@ -60,82 +58,88 @@ private String getScriptOutput() {
...
+        ModifiedNetwork modifiedNetwork = loadNetworkFromProjectCase(baseCase);
 
-                if (baseCase.getScriptError() != null) {
-                    return new ModifiedNetwork(network, baseCase.getScriptError(), baseCase.getScriptOutput());
-                }
+        if (modifiedNetwork.getScriptError() != null) {
+            return modifiedNetwork;
+        }
+
+        // getNetwork script
```
Comment (mathbagu) [inline afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java]: This one too

File: afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java (hunk 1 above)
Comment (geofjamg) [reply afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java]: Done

File: afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java (hunk 2 above)
Comment (geofjamg) [reply afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java]: Removed

File: afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java (hunk 3 above)
Comment (geofjamg) [reply afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/LocalNetworkService.java]: Removed

**Pull request merged**